appserver/tests/test-micro-jar/src/test/java/fish/payara/admin/cli/ListHealthCheckServicesTest.java

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Could these classes be moved to a `healthcheck` subfolder?

Reply:
Yes, you are right. The idea is that more tests are added to this module so they probably should be in a package for bit more structure.

Reply:
Moved.